content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -189,6 +189,15 @@ spec:
 
 This ensures that even pods that aren't selected by any other NetworkPolicy will not be allowed ingress or egress traffic.
 
+## SCTP support
+
+{{< feature-state feature-state state="alpha" >}}
+
+Kubernetes introduces the support of SCTP as a `protocol` value in `NetworkPolicy` definitions as alpha feature. To enable this feature, the cluster administrator needs to enable the `SCTPSupport` feature gate on the apiserver, for example, `“--feature-gates=SCTPSupport=true,...”`. When the feature gate is enabled, users can set the `protocol` field of a `NetworkPolicy` to `“SCTP”`. Kubernetes sets up the network accordingly for the SCTP associations, just like it does for e.g. TCP connections.
```

Comment:
as alpha feature -> as an alpha feature

Reply:
Probably best to remove quote-marks from inside the code blocks. 

`“SCTP”` can just be SCTP if you want to show a string value. The docs style guide talks about this [here](https://kubernetes.io/docs/contribute/style/style-guide/#use-normal-style-for-string-and-integer-field-values).

Reply:
I think you don't need the "e.g." at the end
"just like it does for e.g. TCP connections" -> "just like it does for TCP connections"